scripts/setup-operators.sh

```diff
@@ -28,9 +28,9 @@ install_etcd_operator(){
   kind: Subscription
   metadata:
     name: etcd
-    namespace: ${OPERATOR_HUB_PROJECT}
+    namespace: openshift-operators
```

Comment:
Now operators in the namespace `openshift-operators` will be available globally to any namespace, right?

Reply:
Yes, indeed. :+1: